swervelib/rev/NeoSteerControllerFactoryBuilder.java

```diff
@@ -159,9 +159,9 @@ public void setReferenceAngle(double referenceAngleRadians) {
             // end up getting a good reading. If we reset periodically this won't matter
             // anymore.
             // if (motorEncoder.getVelocity() < ENCODER_RESET_MAX_ANGULAR_VELOCITY) {
```

Comment:
this commented-out logic is the stuff I keep pleading to bring back in. Remember how we sometimes have a module that doesn't get configured properly? I strongly believe it's because this logic is commented out. Prove me wrong

Reply:
Last time we uncommented this, it logged  too much and caused packet loss. We will try again, without some of the logging